lighthouse-core/lib/stack-packs.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const stackPacks = require('@lighthouse/stack-packs');
+
+const stackPacksToInclude = [{
+  packId: 'wordpress',
+  npmOrIconName: 'wordpress',
+  requiredStacks: ['js:wordpress']
+}];
+
+/**
+ * @param {LH.Artifacts} artifacts
+ * @return {Array<LH.StackPack>}
+ */
+function getStackPacks(artifacts) {
+  /** @type {Array<LH.StackPack>} */
+  const packs = [];
+
+  artifacts.Stacks.forEach(pageStack => {
```

Comment:
if you switch to `for (const pageStack of artifacts.Stacks)` you won't need the extra type assertion over `const packs = []`